data/ecosystems/d/defi-tools.toml

```diff
@@ -0,0 +1,6 @@
+title = "DeFi-Tools"
+sub_ecosystems = []
+github_organizations = ["https://github.com/DeFi-Tools"]
```

Comment:
we have a process that finds all repos under orgs and maps those repos to the parent ecosystem. I see that this org has 120+ repos but only `sudograph` should belong to IC. Could you add only that repo to icp? (Instead of this whole ecosystem)